packages/dd-trace/src/platform/node/metrics_lightstep.js

```diff
@@ -9,6 +9,7 @@ const Histogram = require('../../histogram')
 const si = require('systeminformation')
 
 const MICROSECOND = 1 / 1e6
+const NANOSECOND = 1 / 1e9
```

Comment:
If we named this `SECONDS_PER_NANO` it would be easier to understand.

Reply:
hmm a bit strange name tbh as this is 1/1000000  which is definition of nano second